packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -689,6 +689,7 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   Future<void> ensureVisible(
```

Comment:
The doc comment on this function should describe what `padding` does, and under what circumstances you might want to use it.